base/util/PKICertImport.bash

```diff
@@ -0,0 +1,333 @@
+                # Always required; certificate trust flags.
+                CERT_TRUST="$1"
+                shift
+            elif [ "x$arg" == "x--usage" ] || [ "x$arg" == "x-u" ]; then
+                # Always required; certificate usage flags.
+                CERT_USAGE="$1"
+                shift
+            else
+                # We print help whenever the return code is 1, so we don't
+                # need to explicitly parse a --help flag, but we will get
+                # an extraneous but harmless unknown argument message.
+                __e "Unknown argument: $arg"
+                __e "Check your option syntax; perhaps a prior argument is" \
+                     "missing a value?"
+                return 1
+            fi
+        done
+
+        # Ensure that we've seen the required arguments.
+        if [ "x$NSSDB" == "x" ]; then
+            __e "Missing NSS Database location: specify --database/-d"
+            return 1
+        elif [ "x$CERT_PATH" == "x" ]; then
+            __e "Missing certificate location: specify --certificate/-i"
+            return 1
+        elif [ "x$CERT_NICKNAME" == "x" ]; then
+            __e "Missing certificate nickname: specify --nickname/-n"
+            return 1
+        elif [ "x$CERT_TRUST" == "x" ]; then
+            __e "Missing certificate trust: specify --trust/-t"
+            return 1
+        elif [ "x$CERT_USAGE" == "x" ]; then
+            __e "Missing certificate usage: specify --usage/-u"
+            return 1
+        fi
+
+        # All good to go.
+        return 0
+    }
+
+    # Show help and usage information.
+    function _print_help() {
+        if (( $1 != 0 )); then
+            echo ""
+        fi
+
+        echo "Usage: $0 [arguments]"
+        echo "$0 validates and imports certificates."
+        echo ""
+        echo "Requred arguments:"
+        echo "--database, -d <path>: path to the NSS DB"
+        echo "--certificate, -i <path>: path to the certificate to import"
+        echo "--nickname, -n <name>: nickname for the certificate"
+        echo "--trust, -t <flags>: trust flags for the certificate"
+        echo "--usage, -u <flag>: usage flag to verify the certificate with"
+        echo ""
+        echo "Optional arguments:"
+        echo "--ascii, -a: the certificate is in ASCII encoded"
+        echo "--password, -f <path>: password file for the NSS DB"
+        echo "--hsm, -h <name>: name of the HSM to use"
+        echo ""
+        echo "Environment variables:"
+        echo "VERBOSE: see certutil commands being run"
+        echo ""
+        echo "For more information about these options, refer to the" \
+             "certutil documentation."
+    }
+
+    # Import a certificate into the NSS DB specified on $1. When a HSM is
+    # specified and the database is a newer, sqlite format, we run two
+    # commands: one for importing the certificate into the HSM, and one for
```

Comment:
We always use just one command. It is up to the caller to know the trust flags. Update this.